jukebox_radio/users/models.py

```diff
@@ -20,6 +20,9 @@ class User(AbstractUser):
 
     @property
     def spotify_access_token(self):
+        if not self.encrypted_spotify_access_token:
```

Comment:
Line 28 throws an exception, something like `cannot read attribute "encode" of None type object`. So here I catch the `None` case and return `None` if no token exists.